vrk-kpa/suomidigi
# SUOPA-718 Blogs

To test: 
- `make fresh`
- Go to https://suomidigi.docker.sh/blogit and check that the layout is similar to this one: https://xd.adobe.com/view/c1b40caa-b2e0-43a5-78d8-9690a43f540c-b785/screen/0e59f7f9-4fae-4303-ac4b-3d224108c52d/Suomidigi-blogi-eikuvia/
- Also check that mobile and tablet widths looks ok
